.travis.yml

```
@@ -6,30 +6,64 @@ node_js:
   - "10"
 cache: npm
 
-before_install:
-  - npm i -g lerna
-
-install: true
-
-before_script:
-  - export TZ=Europe/Vienna
-
-script:
-  - travis_wait npm ci
-  - lerna run build:release > /dev/null 2>&1
-  - npm test -- --runInBand --ci --bail
-
-before_deploy:
-  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
-
-deploy:
-  - provider: script
-    skip_cleanup: false
-    script: npm i && npm run lerna:publish-next
-    on:
-      branch: develop
-  - provider: script
-    skip_cleanup: false
-    script: npm i && npm run lerna:publish
-    on:
-      tags: true
+
+
+stages:
+  - name: test
+  - name: deploy
+    if: ((branch = develop) OR (tag IS present)) AND (env(TRAVIS_PULL_REQUEST) = false)
```

I just noticed that `env(TRAVIS_PULL_REQUEST )` returns a string so that this condition always returns false

```suggestion
    if: ((branch = develop) OR (tag IS present)) AND type != pull_request
```